public/style.css

```diff
@@ -182,3 +182,185 @@ input.nav_search_box{
+	text-decoration: none;         
+	font-size: 21px;         
+	font-weight: initial;     
+	color: var(--darkgrey);
+	font-style: normal; 
+	padding-bottom: 28px;
+	position: relative;
+	left: 3%;
+	width: 49%;
+	margin-top: 23px;
+	margin-bottom: 16px;
+	border: none;
+	outline: none;
+}
+
+.tags-par{
+	position: relative;
+	font-style: normal;
+	font-size: 16px;
+	font-weight: 600;
+	color: black;
+	left: 3%;
+}
+
+.study_cards{
+	position: relative;
+	bottom: 29px;
+}
+
+.edit-quiz-tags-input{
+	position: relative;
+	background-color: rgb(223, 223, 223);
+	left: 3%;
+	margin-bottom: 15px;
+	bottom: 16px;
+	height: 42px;
+	width: 34%
+}
+
+#edit-tags-input{
+	position: absolute;
+	outline: none;
+	font-style: normal;
+	font-weight: normal;
+	font-size: 17px;
+	border: none;
+	background-color: rgb(223, 223, 223);
+	width: 100%;
+	height: 100%;
+}
+
+.edit-questions{
+	font-size: 21px;
+	font-weight: 600;
+	font-style: normal;
+	margin-top: -1px;
+	margin-bottom: -17px;
+	position: relative;
+	left: 3%;
+	color: black;
+}
+
+.edit-questions-container{
+	background-color: rgb(223, 223, 223);
+	position: relative;
+	resize: none;
+	outline: none;
+	left: .5%;
+	width: 99%;
+}
+
+.edit_description{
+	word-wrap: break-word;
+	white-space: pre-wrap;
+	height: 46px;
+}
+
+#quiz-description-input{
+	background-color: rgb(223, 223, 223);
+	font-size: 17px;
+	font-weight: normal;
+	font-style: normal;
+	resize: none;
+	overflow: hidden;
+	border: none;
+	display: inline-block;
+	position: absolute;
+	left: 3%;
+	margin-top: -21px;
+	width: 34%;
+	height: 40px;
+	outline: none;
+}
+
+.new-flash-card #edit-quiz{
+	background-color: white;
+	color: red;
+	position: relative;
+	left: 2px;
+	bottom: 8px;
+}
+
+.new-flash-card{
+	background-color: white;
+	border-radius: 6px;
+	position: relative;
+	left: 22px;
+	width: 94.4%;
+	margin: 17px;
+	top: 16px;
+}
+
+#quiz-question-input{
+	position: relative;
+	left: 0px;
+	top: 3px;
+	font-size: 15px;
+	height: 23px;
+	resize: none;
+	width: 47%;
+	background-color: rgb(223, 223, 223);
+	border-color: white;
+	outline: none;
+}
+
+#quiz-answer-input{
+	position: relative;
+	left: 2%;
+	width: 48%;
+	font-size: 15px;
+	resize: none;
+	background-color: rgb(223, 223, 223);
+	border-color: white;
+	height: 23px;
+	top: 3px;
+	outline: none;
+}
+
+.add-new-card #edit-quiz{
+	border-style: dotted;
+	border-color: var(--darkgrey);
+	border-width: 2px;
+	background-color: rgb(223, 223, 223);
+	text-align: center;
+	width: 96%;
+	text-decoration: none;
+	font-size: 20px;
+	color: var(--darkgrey);
+	position: relative;
+	left: 1.2%;
+	margin-top: 17px;
+	margin-bottom: 2px;
+}
+
+.finish-new-quiz #edit-quiz{
+	background-color: var(--brand-color);
+	color: var(--light-text);
+	position: relative;
+	font-size: 15px;
+	left: 85%;
+	width: 6%;
+	height: 28px;
+	margin-bottom: 15px;
+}
+
+.add-new-card{
+	position: relative;
+	margin: 17px;
+}
+
+.finish-new-quiz{
+	position: relative;
+	margin: 17px;
+}
+
+#edit-quiz:hover{
```

Comment:
I think we might want to make a general “clickable” class for all buttons so we can reuse this css for other buttons.